### PR TITLE
Home page: What is sus portion

### PR DESCRIPTION
**Pull request by GDivino**
- [X]  Desktop view

- [X]  Tablet view

- [X] Mobile view

Notes
- Changed App.vue for testing purposes
- Created a temporary button while there's no button component yet.

### CODE REVIEW

**Comment by GDivino** [comment]
Relevant screenshots:
- Desktop view:
image
- Tablet view:
image
- Mobile view:
image

File: src/components/Home/WhatIsSus.vue [hunk 1]
```diff
@@ -0,0 +1,128 @@
+<template>
+    <div class="what-is-sus">
+        <div class="text">
+            <h1>What is Startup Summit?</h1>
+            <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.</p>
```
Comment (Znarfois) [inline src/components/Home/WhatIsSus.vue:5]: We need to fill this nalang with the actual content before we merge.

File: src/components/Home/WhatIsSus.vue [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+            <button class="learn-more" type="button">Learn more</button>
```
Comment (Znarfois) [inline src/components/Home/WhatIsSus.vue:6]: Replace with the button from Clarz

File: src/components/Home/WhatIsSus.vue [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+        </div>
+        <div class="picture"></div>
```
Comment (Znarfois) [inline src/components/Home/WhatIsSus.vue:8]: Let's add the picture first before merging. Make sure to add an alt = "something"

**Review by Znarfois — CHANGES_REQUESTED** [review]
Address the comments nalang, then we can merge!

**Pull request closed without merging**